build_tools/cmake/build_benchmarks.sh

```diff
@@ -33,8 +33,8 @@ echo "Configuring to build benchmarks"
 "${CMAKE_BIN}" -B "${BUILD_BENCHMARKS_DIR}" \
```

Comment:
Here for threading:

Did you meant to include "Fixes" pointing at a pull request?

Reply:
Realized it's not an issue. Filed and pointed to a new one #10988